CONTRIBUTING.adoc

```diff
@@ -0,0 +1,108 @@
+= How to contribute to Rudder
+
+Rudder is open to all kinds of contribution:
+
+* Bug reports and fixes
+* Documentation
+* Features ideas
+* New features
+
+If you want to contribute, the best way to start is to come and chat with us
+on https://chat.rudder.io[chat.rudder.io]. You can also communicate by email
+using the  https://www.rudder-project.org/mailman/listinfo/rudder-dev[rudder-dev]
+mailing list.
+
+NOTE: Contribution to Rudder requires a https://www.rudder.io/en/expand/contribute/#panel-2422-8-0-1[CLA/CCLA].
+
+== Development workflow
+
+We maintain several Rudder major versions in separate branches.
+
+* `branches/rudder/5.0` is Rudder 5.0, from which 5.0 minor release will be tagged
+* `master` is the next Rudder version, not branched yet
+
+NOTE: Some tooling repositories only have a `master` branch.
+
+All changes need to be done in the oldest relevant branch (i.e.
+affected and still maintained).
+Version status is visible on https://docs.rudder.io/version.
+
+We have an internal CI tooling, that we are intend to integrate to Github pull
+requests status.
+
+=== Issue tracker
+
+We do not use Github's issue tracking but an external issue tracker hosted on
+https://issues.rudder.io.
+
+Every change in one of the Rudder repositories must be linked to an issue in this
+tool, except for documentation fixes that can be contributed directly on
+Github.
+
+=== Development tooling (rudder-dev)
+
+==== Installation
+
+To ease the use of a separate issue tracker and maintenance of several
+branches of Rudder, we have developed a dedicated tooling.
+
+To download rudder-dev:
+
+```
+wget https://repository.rudder.io/tools/rudder-dev
+```
+
+It is a shell script you can put somewhere is your PATH. You can then update it with
+
+```
+rudder-dev update
+```
+
```

Comment:
You should tell here that there's a configuration file to update with github token.